openlibrary/solr/updater/author.py

```diff
@@ -10,23 +15,35 @@ class AuthorSolrUpdater(AbstractSolrUpdater):
     async def update_key(self, author: dict) -> tuple[SolrUpdateRequest, list[str]]:
         author_id = author['key'].split("/")[-1]
         facet_fields = ['subject', 'time', 'person', 'place']
```

Comment:
Move this out up above as a global constant, then we can use it in the `top_subjects` method to avoid the `dict` sniffing which could be error prone if we add other facets in the future.

```suggestion
        SUBJECT_FACETS = ['subject_facet', 'time_facet', 'person_facet', 'place_facet']
```